### DISCUSSION TITLE
HTTP Error 400: Bad Request

### DISCUSSION THREAD

**Discussion by IndrajeethY** [deta/cloud-docs#511] [Help] [answered]
def gban_user(self, userId, reason, timestamp, qid):
        try:
            value = {"key": userId, "reason": reason, "timestamp": timestamp, "qid": qid}
            self.gbanned.put(value)
            return True
        except Exception as e:
            print(e)
            return e
            
            y is this error?

**Comment by IndrajeethY** [comment, marked as answer]
nvm found reasaon

**Reply by timveph** [reply]
What was the reason?
